--- libs/celestial.js-orig	2018-11-30 13:20:47.632758430 +0000
+++ libs/celestial.js	2018-11-30 13:28:57.966220021 +0000
@@ -11,7 +11,7 @@
     ANIMINTERVAL_R = 2000, // Rotation duration scale in ms
     ANIMINTERVAL_P = 2500, // Projection duration in ms
     ANIMINTERVAL_Z = 1500, // Zoom duration scale in ms
-    ZOOMEXTENT = 10;        // Maximum extent of zoom (max/min)
+    ZOOMEXTENT = 50;        // Maximum extent of zoom (max/min)
     
 var cfg, prjMap, zoom, map, circle;
 
@@ -1059,8 +1059,8 @@
   //  with size,shape,color: "prop=val:result;.." || function(prop) { .. return res; } 
   if (!has(dat, "type")) return console.log("Missing type");
   
-  if ((dat.type === "dso" || dat.type === "json") && (!has(dat, "file") || !has(dat, "callback"))) return console.log("Can't add data file");
-  if ((dat.type === "line" || dat.type === "raw") && !has(dat, "callback")) return console.log("Can't add line");
+  if ((dat.type === "dso" || dat.type === "json") && (!has(dat, "file") && !has(dat, "callback"))) return console.log("Can't add data file", dat);
+  if ((dat.type === "line" || dat.type === "raw") && !has(dat, "callback")) return console.log("Can't add line", dat);
   
   if (has(dat, "file")) res.file = dat.file;
   res.type = dat.type;
@@ -2625,6 +2625,10 @@
     return kepler;
   };
 
+  Object.defineProperty(kepler, 'name', {
+    writable: true,
+  });
+
   kepler.name = function(_) {
     if (!arguments.length) return name; 
     name = _;
